megamek/src/megamek/server/Server.java

```diff
@@ -2887,9 +2832,7 @@ private void endCurrentPhase() {
                 changePhase(IGame.Phase.PHASE_EXCHANGE);
                 break;
             case PHASE_EXCHANGE:
-                game.addReports(vPhaseReport);
-                changePhase(IGame.Phase.PHASE_SET_ARTYAUTOHITHEXES);
-                break;
+                // TODO : Windchild this might have been a bug
```

Comment:
Is this change correct or was it a bug that it was identical to the next Phase?

Reply:
This is correct. In a regular game, PHASE_SET_ARTYAUTOHITHEXES comes after PHASE_EXCHANGE. In a scenario it comes after PHASE_STARTING_SCENARIO.